marketplace/optimizely/test/AppPage.spec.js

```diff
@@ -1,40 +1,26 @@
 import React from 'react';
-import { cleanup, render, wait } from '@testing-library/react';
+import { cleanup, render, wait, fireEvent, act } from '@testing-library/react';
 
 import AppPage from '../src/AppPage';
 import mockProps from './mockProps';
 import projectData from './mockData/project.json';
 import contentTypesData from './mockData/contentTypes.json';
 
 const basicProps = {
-  openAuth: jest.fn(),
+  openAuth: () => {},
   accessToken: '',
-  client: {},
+  client: {
+    getProjects: () => Promise.resolve(projectData)
+  },
   sdk: mockProps.sdk
 };
 
 describe('AppPage', () => {
   afterEach(cleanup);
-
-  it('should render the AppPage with Features and connect button', () => {
-    const { container } = render(<AppPage {...basicProps} />);
-
-    expect(container).toMatchSnapshot();
-  });
-
-  it('should render the AppPage loading when first connected to Optimizely', () => {
-    const props = { ...basicProps, accessToken: '123' };
-
-    const { container } = render(<AppPage {...props} />);
-
-    expect(container).toMatchSnapshot();
-  });
-
-  it.skip('should render the Optimizely config when loaded', async () => {
-    const client = {
-      getProjects: () => Promise.resolve(projectData)
-    };
-
+  // There is potentially a bug with react/testing library here. The order that these tests run
```

Comment:
Please refer in the comment to the library as `@testing-library/react`. It's easier for a search.

Reply:
Done here: 7b46189f64638460de60475e54bdd10bd3933e27